docs/explanations/what_is_a_fixable_vulnerability.rst

```diff
@@ -0,0 +1,7 @@
+What is a fixable vulnerability ?
+*********************************
+
+On any ``pro vulnerability``, we consider a vulnerability (CVE or USN) as **fixable**,
+if and only if, at least **one** affected installed package associated with it can
+be fixed. That means that a vulnerability is unfixable if no affected installed packages
+related to the vulnerability can be fixed at the moment.
```

Comment:
This is really too short to warrant its own page. Can we move this content to the explanation landing page instead, and show it ahead of the links to the list/show pages as a sort of brief introduction to the concept of fixable vulnerabilities?

Reply:
That's a good idea